script/analysis/plot.py

```diff
@@ -329,29 +329,39 @@ def overlay_field(ax, geom, dump, NLEV=20, linestyle='-', linewidth=1,
   linecolor='k'):
   from scipy.integrate import trapz
   hdr = dump['hdr']
-  N1 = hdr['N1']; N2 = hdr['N2']
-  x = flatten_xz(geom['x'], hdr).transpose()
-  z = flatten_xz(geom['z'], hdr).transpose()
+  N1 = hdr['N1']//2; N2 = hdr['N2']
```

Comment:
Why are you dividing `hdr['N1']` by 2?

Reply:
Yes, this is the line that I wanted to check with you. 

Is `A_phi[j,N1-1-i]` calculating A in the -ve direction and `A_phi[j,i+N1]` in the +ve direction? So if I don't divide by 2, ultimately, in line 363, `rcyl` and `z` ends up having dimensions 256 X 256, and `A_phi` is 256 X 512. So I divided by 2 to make N1 represent cells in either the +ve/-ve direction only.